src/pages/notes/document/DocumentNotePage.tsx

```diff
@@ -65,10 +79,16 @@ const DocumentNotePage: React.FC = () => {
 
     useEffect(() => {
         const receiveMessage = (event: any) => {
-            console.log('Received message from iframe:', event.data);
             if (event.data === 'CONFIRMED') {
                 setConfirmed(true);
             }
+            else if (event.data.startsWith('SAVE ')) {
+                const content = event.data.substring(5);
```

Comment:
why substring 5?

Reply:
Because we are sending the string message in the format `SAVE <content>` to the main applicaction so to extract the real content we have to cut off the phrase `SAVE ` from beggining